Align GCP infrastructure manifest to wordpress profile

ody opened the pull request:
This commit aligns some style best practice with the accompanying
  wordpress profiles and makes one SQL user change that will enable
  Puppet to manage databases.

nelsonjr commented:
I split the file into various steps because of dependencies that cannot be resolved by functions. I can help merge this for you if you wish.

ody committed:
Move control directory around and add Puppetfile

ody committed:
Update default db_host value

  This will enable Class['wordpress'] to look up the database target
  from the GCE instance's metadata but leaves it override-able for use in
  EC2 instance.
